mqtt/mqtt-bridge/src/bridge.rs

```diff
@@ -74,11 +88,17 @@ impl Bridge {
             .into_iter()
             .map(|topic| topic.try_into())
             .collect::<Result<Vec<_>, _>>()?;
+
+        // component_sender can be used for configuration changes
+        let (connectivity_sender, connectivity_receiver) =
+            mpsc::unbounded_channel::<BridgeMessage>();
```

Comment:
not sure that we want unbounded channel here. what do you think?

Reply:
addressed in master PR